index.js

```diff
@@ -59,8 +59,8 @@ module.exports = customService(async function index(service) {
   service.decorate('GREETING_TYPE', {
     HELLO: 'hello',
   })
-  service.addRawCustomPlugin('POST', '/greetings', async function handler(req, reply) {
-    const currentUserId = req.getUserId()
+  service.addRawCustomPlugin('POST', '/greetings/:userId', async function handler(req, reply) {
```

Comment:
Remove the concept of `userId` from this handler.

We could call it `from` and `to` (instead of "who") get both of them from body

Reply:
I agree with dibericky, modify the body to feature a new property (such as `from` and use it instead of the userId).

We can keep `who`, no need to change it too, imho